vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/events/HasItem.java

```diff
@@ -27,18 +32,75 @@
     String getCategory();
 
     /**
-     * Returns the index of {@link #getItem()} in {@link #getSeries()}.
+     * Returns the index of the series item, that is associated with this event,
+     * in {@link #getSeries()}. Can be used to identify the item within the
+     * series.
+     * <p>
+     * Example for {@link ListSeries}:
      *
-     * @return
+     * <pre>
+     * int itemIndex = event.getItemIndex();
+     * ListSeries series = (ListSeries) event.getSeries();
+     * Number datum = series.getData()[itemIndex];
+     * </pre>
+     *
+     * @return the index of the item in the series
+     * @see #getItem()
+     * @see #getItemId()
      */
     int getItemIndex();
 
     /**
-     * Returns the item that was clicked
+     * The ID of the series item that is associated with the event. Can be used
+     * to identify the item within the series.
+     * <p>
+     * Example for {@link NodeSeries}:
+     *
+     * <pre>
+     * String id = this.getItemId();
+     * NodeSeries series = (NodeSeries) this.getSeries();
+     * Optional&lt;Node&gt; nodeForId = series.getNodes().stream()
```

Comment:
This line shows correctly in the IDE, but it is shown different than other places.

Reply:
Can you be more specific what you mean by different, and other places? JavaDoc is basically HTML, so if we want the `<Node>` char sequence to show up correctly, then the characters have to be escaped, otherwise they will be interpreted as HTML tag.

Reply:
Like the line above where we have `<pre>` tag.